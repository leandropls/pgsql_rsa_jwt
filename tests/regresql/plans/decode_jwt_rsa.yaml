rs256:
  jwk: '{"n": "qgfVLnNNaLLoro-7f4y83PN-W78CnF5qHj0tH72tQGk8NDvDlB5uil8f2JZXrT6wp6r3rKXF5Cm_RGOHm0CMM8607IWL5VBkecP7MiOGacTg44NMXe0Dcf2kLcuvJLpX7VerE6SPBWYe4pTOoGPRugb-1dF52Kc9e9Qni9iIgSprTOK23_JEokX573LYTiANGbsMAAyvIRCMGAShp15AeqWn1-7_EIB37b6PJAkbMPN5PHWa_wF8dPSJEOYSPoOPQoxN306-ZXnNx6JF87SyQQmEeoMjhzlASUP1KuMtlQuNHfQkW08MlJvyFdyweAkCeXdL8hAaTW4FiW_klJN0NQ", "e": "AQAB", "alg": "RS256", "use": "sig", "kid": "7588689e-63ce-4abb-9906-8797e09c8928", "kty": "RSA"}'
  token: eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6Ijc1ODg2ODllLTYzY2UtNGFiYi05OTA2LTg3OTdlMDljODkyOCJ9.eyJhdWQiOiJwb3N0Z3Jlc3FsIn0.ddCxuCbIOGwU760UJPWEKbJiCSvs8gONDdut78C5nMR43_OlbtGxL9ZOiOABjleMF9eQMPciRy0ykVOTkhLu4XohQ9f4Ja0tQxABcb6X19V0Ozw1joToR7H4AuGzEowuZsTOBU2LCauiZkSLIZX8cbY5jy0ITGleZuP2CdpZUDATR7pQaQ0dTma6GyPAOWNcBtP3dVgpMDnjvqYnSz_Phtq9HJWIT-OWq2j_Qm9UMO1EmrHIzjmoI4_2fMhLHxt9000THvp3gwgQY67luectLdQDcum6tk-kOtJyEWsaJkYlrgRcNiupZJ2i8IjIFOqGiU3dRF9rK32EQ2SUBUjEMw
rs384:
  jwk: '{"n": "owY4zP3NlsCPeqnZr4OgCUCN8NOal9lD22u51Qkn_T6Zrl3VmDuo7WoYPojwPh1cycu2_UJf_0ZZBjkvaqS4O0kezrG0hzn_Bi-tf_hbQMHd0bcFyfS7LoDgFdg4788rYXPUDtw5-EmpvOhbaT4g7-D8xe8Z-qd73t40FZdUJFTJ54ylzSXZCtjAhC1kSiAiI48vXbpHZebdbbcrCHYYZENAtO4Dwf5a2X6tIpD9wFkQGZhYhGqh0QihEE_xheLxsBs4NHRZHAqZhuHiAJrEk2xZuS1i8i3R_QSkVz5BdAvPv0DBfxzTFYQr5MiRY2hrkyoMRLL_3G2VPXLwC0svMA3PTP5G9jYpv-nXMOU1r2agOaZvAWvEN64acRCbGFBE4aIVrrvWnJAEEcQqUyhazzv21k6-2erkOZSYNwzzgOM6gpYUZH7L_bAK2Xp1S_f2PRUNHopNGpdEfUJKwcGR9won56I-5PN_tjG-8pw9b3CnckMDUNaacgRGbCN6stMN", "e": "AQAB", "alg": "RS384", "use": "sig", "kid": "bc51ed0b-9ccb-4500-89c1-e7163e295481", "kty": "RSA"}'
  token: eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzM4NCIsImtpZCI6ImJjNTFlZDBiLTljY2ItNDUwMC04OWMxLWU3MTYzZTI5NTQ4MSJ9.eyJhdWQiOiJwb3N0Z3Jlc3FsIn0.WAUrQVifK9yFCA7hTuMcn3aJ6aBf2Uj1dMjS7AGr11GCtYL7K7bQJwxwmDPHDLj5MaWLL1ugjGrf5PDxRzuFmiu1yZ-MI-ldWTHWmcQV1VcUUS4Ui8mdAXbzjuxRoqKvVPv9eCqcAUDSTe3tJio59jUGaEfG1PpABBkk-kU6u6Arfen5hIlEIAPP1sjP8MVs_w_SJYHuQSJtkLlPpP9gTrk9x2cEEzr5zOxkCyM5thVbEtO7resxWrT6k52KZaDmB0KSiAhUC2kmErVDIWbFpGQiYVZd-HhiCysPi49_Wvjhv7vCJmKDSeC2FwAXROG1C92KIA5svWDOhdytFV_vWOtV9Re0uIUodiA4kLeqtuHtp2QKRWz3OwXUFaF7vkp9d_d7pzllwmm3XUc4DF4EuxIYfKVx8p7Hc7vYGC0yue4T4S2grQz_MqiqZHmP0sP0ecVN1pGtuDkxMfWxE1rJ9n-u8pCmP6kX7JC-rgMjBcNEyaqHNv9yyIJRjIN92-53
rs512:
  jwk: '{"n": "tKfcLT6V0RxeyW1z3aZldnj6W-pmW_WX4qaXPSCicD-PChs0ehGkL5q1yW-hANrwwrapy9YbgmvFLbCyRXwrnT5M-W45FsZa5bZxrpOCk_hWOQTrmJRsehlksZp9raBNn3tf77m0h1SuHuzl6LoGGb3kC1dDNBCSUzyRwsgKxV-pAPgbdB7Md--1tuN3sJ_M4i0WVFX2Hy34Kit8-u_KP1rNzoAz7CXjYloohVUy-hX0cCb1R5m73Ki-1jj4Ttl0bIBf84Wcs2a1qMoUBiQD2dvjfgc__1f8sdWQCeR42BSDPV4ZqqZCBUTR6AE7w6eSoDxeMNEMqSqMjyKTCnMsVK427YoqJ5U9as-Xuuc3kGQXsuB_WNnlMvISz8lJ5HSRR_WaIvwfpGVB58bBAu5ayuXn7UUqcu-dZNTZ4vA3fDYTjjNVBqve0jGenUVKujcrs-zNH2cAotBPLu5POmFkv-9v95YgT4ZO4Ao7T6b0zNBVvX8fZnzf-Kzpa7hRMrvjmfn2xJs1hKUMxJx1xEGBMZEDVBu_lqxAljXNB2flL-xyahg-EtBM3mTErvIjaEq2JMX8HcBvVA-MsgW9sYXKjlPfQL6wD0uO_Bi15_Y_uqNva9dYvXWPfWBtqnC8iseyyNJ_4aV4IlWHLQC9RxVrUmOK-gQYkBKfKQZe2HCpXbs", "e": "AQAB", "alg": "RS512", "use": "sig", "kid": "527efc33-4fd9-4719-9e95-ecd098eabd73", "kty": "RSA"}'
  token: eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzUxMiIsImtpZCI6IjUyN2VmYzMzLTRmZDktNDcxOS05ZTk1LWVjZDA5OGVhYmQ3MyJ9.eyJhdWQiOiJwb3N0Z3Jlc3FsIn0.ni-DipoGmjoeNYHL-2XBz9W3mUTmPoZ0pyBPwDZxePX00L7a7gnb_GJU24Di59bpw4vQ1glolnSoEL9C96VpLGvW-ughgzHh9ha-3Si7oMZB_y470HNMHwCujMqSEUy6Zp3Iv_0dQGJBeZ85MOgIi6G_eDzZzWAbUUTf5Ad0d3rbVAG3gjFXIRPKHS3N9BxqpcjJ7CwbVHNFQH890CI6M2phvSVBNglx-geQKA1Z8U-LTZe4IZgJj7-6iE0NkfXC8I-fleIb-YctJCZYqW4duArnKPNDGhz1dIEZvnOR-UqEsW3HGdMs4uwnX3Qxc-Zf8fEZdsH3XvGIBT8MY1k0iviIhLqUp_ZOQYdqJrLn11iXTf_xhx-EMYRwiaWcN2-epooVp_JqrfCjp9yhSRG_TlsfgG6O57XNynU7Jhxti4UZg5X9Ix2NRC8tLjQEcyKV3-zi314dQSWHIFSqnGfegci1bSgVGNZ43-chMnezlJHTjnXtLuZ_9IP7JrJUrHGBhcv9_5TYfWPgqHP1DPTUHw3GNu_9mLZg3AV0L7EY14oJh2b13n20xZeTidS808su8sJMB4DqkSPYlv82NWpSkANa-ygktuCVe_tXnSp7J65TMVgvr7lX_XH2OOl2BT8AhuodR2g3GldteKS8fvId_-wvqvhgfEBQFTfYvCe5yXA
