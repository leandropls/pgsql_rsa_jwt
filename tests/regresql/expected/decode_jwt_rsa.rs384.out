   decode_jwt_rsa    
---------------------
{"aud": "postgresql"}
